website/www/site/content/en/get-started/beam-overview.md

```diff
@@ -29,7 +29,6 @@ Beam is particularly useful for [embarrassingly parallel](https://en.wikipedia.o
   <img src="/images/learner_graph.png" width="800px" alt="Learner Graph">
 </div>
 
-
```

Comment:
nit: did you intend to remove this line?

Reply:
Probably from the formatter. Is this line needed?